src/main/java/net/sf/jabref/gui/entryeditor/FileAnnotationTab.java

```diff
@@ -66,78 +59,73 @@
     private final JScrollPane highlightScrollPane = new JScrollPane();
     private final JButton copyToClipboardButton = new JButton();
     private final JButton reloadAnnotationsButton = new JButton();
-    DefaultListModel<FileAnnotation> listModel;
+    private DefaultListModel<FileAnnotation> listModel;
 
     private final EntryEditor parent;
-    private final BasePanel basePanel;
-    private final JTabbedPane tabbed;
-    private int commentListSelectedIndex = 0;
 
+    private Map<String, List<FileAnnotation>> annotationsOfFiles;
     private boolean isInitialized;
 
-    private Map<String, List<FileAnnotation>> allNotes = new HashMap<>();
 
-
-    public FileAnnotationTab(EntryEditor parent, BasePanel basePanel, JTabbedPane tabbed) {
+    FileAnnotationTab(EntryEditor parent) {
         this.parent = parent;
-        this.basePanel = basePanel;
-        this.tabbed = tabbed;
         setLayout(new BoxLayout(this, BoxLayout.X_AXIS));
         listModel  = new DefaultListModel<>();
         this.isInitialized = false;
 
     }
 
-    public static FileAnnotationTab initializeTab(FileAnnotationTab tab, Optional<Map<String, List<FileAnnotation>>> notes){
+    FileAnnotationTab initializeTab(FileAnnotationTab tab) {
+        if (tab.isInitialized) {
+            return tab;
+        }
+
+        tab.setUpGui();
+        tab.isInitialized = true;
+        tab.parent.repaint();
+        return tab;
+    }
+
+    FileAnnotationTab initializeTab(FileAnnotationTab tab, Map<String, List<FileAnnotation>> cachedFileAnnotations) {
+        this.annotationsOfFiles = cachedFileAnnotations;
+
+        if (tab.isInitialized) {
+            return tab;
+        }
 
-        if(!tab.isInitialized) {
-            
             try {
-                tab.addComments(notes);
+                tab.addAnnotations();
             } catch (IOException e) {
                 e.printStackTrace();
             }
-            tab.setUpGui();
 
+            tab.setUpGui();
             tab.isInitialized = true;
             tab.parent.repaint();
             return tab;
-        }
-        return tab;
+
     }
 
     /**
-     * Adds pdf comments from all attached pdf files belonging to the entry selected in the main table and
-     * shows those from the first file in the comments tab
+     * Adds pdf annotations from all attached pdf files belonging to the entry selected in the main table and
+     * shows those from the first file in the file annotations tab
      * @throws IOException
```

Comment:
I would remove that, `@throws` javadoc, because it does not add any more information.